test/basics.el

```diff
@@ -3,7 +3,7 @@
 
 (ert-deftest eldev-just-run-1 ()
   (eldev--test-run "empty-project" ()
-    (should (string-prefix-p (eldev--test-in-project-environment (eldev--test-capture-output (eldev-usage))) stdout))
+    (should (string-prefix-p (eldev--test-in-project-environment (eldev--test-capture-output (eldev-usage))) (eldev--test-drop-usage-bat-extension stdout)))
```

Comment:
Wouldn't it be easier to adjust `eldev--test-in-project-environment` so that it also produces a string with `.bat` on Windows? Then you wouldn't need to tweak each test separately.

Reply:
Moved the logic to canonicalise usage's bin/eldev path inside  `test/common.el:eldev--test-in-project-environment`.

Reply:
Is it not possible to alter this line: `(eldev-shell-command eldev--test-shell-command))` so that it produces output with the expected script name from get go?

Reply:
Done.